arlas-rest/src/main/java/io/arlas/server/rest/explore/aggregate/AggregateRESTService.java

```diff
@@ -222,13 +233,13 @@ public Response aggregatePost(
         request.basicRequest = aggregationsRequest;
         request.headerRequest = aggregationsRequestHeader;
 
-        AggregationResponse aggregationResponse = getArlasAggregation(request, collectionReference);
+        AggregationResponse aggregationResponse = getArlasAggregation(request, collectionReference, (aggregationsRequest.form != null && aggregationsRequest.form.flat != null && aggregationsRequest.form.flat));
         aggregationResponse.totalTime = TimeUnit.NANOSECONDS.toMillis(System.nanoTime() - startArlasTime);
 
```

Comment:
you could use Boolean.TRUE.equals(aggregationsRequest.form.flat)

Reply:
Done

Reply:
👍